aws.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  backend "s3" {
+    bucket         = "aeternity-terraform-states"
+    key            = "ae-testnet-api.tfstate"
+    region         = "us-east-1"
+    dynamodb_table = "terraform-lock-table"
+  }
+}
+
+provider "aws" {
+  version = "1.55"
+  region  = "us-east-1"
```

Comment:
I assume this is needed because of the s3 backend above ? Did you tried without it?

Reply:
I think it is because of gatway module
```
module "aws_gateway": cannot pass non-existent provider "aws.us-east-1"